modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/meta/IndexMeta.java

```diff
@@ -31,15 +31,21 @@ public class IndexMeta {
     @IgniteToStringExclude
     private final long metaPageId;
 
+    private final int inlineSize;
+
     /**
      * Constructor.
      *
      * @param id Index ID.
      * @param metaPageId Index tree meta page ID.
+     * @param inlineSize Inline size in bytes.
      */
-    public IndexMeta(UUID id, long metaPageId) {
+    public IndexMeta(UUID id, long metaPageId, int inlineSize) {
+        assert inlineSize >= 0 && inlineSize <= 0xFFFF : inlineSize;
```

Comment:
We can make `0` prohibited and use this page type for meta page, what do you think?
Making meta IO type and inner IO type closer to each other would help visually, you know

Reply:
We decided to leave it as is.